XenAdmin/Dialogs/OptionsDialog.Designer.cs

```diff
@@ -59,8 +62,31 @@ private void InitializeComponent()
             // 
             // verticalTabs
             // 
-            resources.ApplyResources(this.verticalTabs, "verticalTabs");
             this.verticalTabs.Items.AddRange(new object[] {
+            this.confirmationOptionsPage1,

```

Comment:
It seems additional copies of the tabs were added automatically to this list when work took place on the designer (you can see them on the left hand side panel if you run the application and bring up the Options dialog). I suspect that this has to do with the logic in the event handlers of ControlAdded/Removed in the VerticallyTabbedDialog, in all likelihood the check of whether the tab exists already is wrong. For the current PR let's just revert this addition. The VerticallyTabbedDialog should be fixed to prevent it from happening again, but feel free to put that commit in a different PR since it is not related to the tooltip (which now seems to work ok).